pwndbg/dbg/lldb.py

```diff
@@ -9,22 +13,363 @@
 import pwndbg
 
 
+class LLDBFrame(pwndbg.dbg_mod.Frame):
+    def __init__(self, inner: lldb.SBFrame):
+        self.inner = inner
+
+    @override
+    def evaluate_expression(self, expression: str) -> pwndbg.dbg_mod.Value:
+        value = self.inner.EvaluateExpression(expression)
+        opt_out = _is_optimized_out(value)
+
+        if not value.error.Success() and not opt_out:
+            raise pwndbg.dbg_mod.Error(value.error.description)
+
+        return LLDBValue(value)
+
+
+def map_type_code(type: lldb.SBType) -> pwndbg.dbg_mod.TypeCode:
+    """
+    Determines the type code of a given LLDB SBType.
+    """
+    c = type.GetTypeCode()
+    f = type.GetTypeFlags()
```

Comment:
move ` f = ...` after the ifs with TypeCode.

Reply:
Alright